src/fileops.rs

```diff
@@ -0,0 +1,21 @@
+extern crate libc;
+
+use std::ffi::CString;
+use libc::c_int;
+
+pub fn open<T: Into<String>>(path: T, mode: c_int) -> c_int {
+  unsafe { libc::open(CString::new(path.into())
+                      .unwrap_or(CString::new("").unwrap()).as_ptr(), mode) }
+}
+
+pub fn close(fd: c_int) -> c_int {
+  unsafe { libc::close(fd) }
+}
+
+#[test]
+fn open_close_devnull() -> () {
```

Comment:
ok. just testing open/close on a file that *should* be in most linux based OS, at least the ones supported by the driver  i need to use and by travis.